src/main/resources/ca/on/oicr/gsi/somnus/ui.js

```diff
@@ -16,21 +23,27 @@ export default function(form, environments, services) {
   ttl.type = "number";
   ttl.value = "1";
   const units = document.createElement("SELECT");
-  for (const [name, multiplier] of [
-    ["days", 86400],
-    ["hours", 3600],
-    ["minutes", 60],
-    ["seconds", 1]
+  for (const [name, value] of [
+    ["minutes", "minutes"],
+    ["hours", "hours"],
+    ["days", "days"],
+    ["seconds", "seconds"]
   ]) {
     const option = document.createElement("OPTION");
     option.text = name;
-    option.value = multiplier;
+    option.value = value;
     units.appendChild(option);
   }
-  units.selectedIndex = 1;
+  units.selectedIndex = 0;
+  units.addEventListener("input", calculateWarning);
+  ttl.addEventListener("input", calculateWarning);
   form.appendChild(document.createTextNode("Time-to-quiet: "));
   form.appendChild(ttl);
   form.appendChild(units);
+  var warning = document.createElement("span");
```

Comment:
`const`

Same everywhere else you've used `var` - cases where `var` is preferable to `const` or `let` are very rare